packages/clarity-js/src/data/metadata.ts

```diff
@@ -243,16 +243,54 @@ function getCookie(key: string): string {
       for (let i = 0; i < cookies.length; i++) {
         let pair: string[] = cookies[i].split(Constant.Equals);
         if (pair.length > 1 && pair[0] && pair[0].trim() === key) {
-          return pair[1];
+          // Some browsers automatically url encode cookie values if they are not url encoded.
+          // We therefore encode and decode cookie values ourselves.
+          return decodeCookieValue(pair[1]);
         }
       }
     }
   }
   return null;
 }
 
+function decodeCookieValue(value: string): string {
+  // For backwards compatability we need to consider 3 cases:
+  // * Cookie was previously not encoded by Clarity and browser did not encode it
+  // * Cookie was previously not encoded by Clarity and browser encoded it once or more
+  // * Cookie was previously encoded by Clarity and browser did not encode it
+  let [isEncoded, decodedValue] = valueIsEncoded(value);
+  if (!isEncoded) {
+    return value;
+  }
```

Comment:
we can save some bytes by cutting lines 262-264, while loop will be skipped and decodedValue would be returned